# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/fastfetch/detect-t2.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- fastfetch-2.8.6/src/logo/builtin.c.vanilla	2024-05-02 09:29:19.407756329 +0200
+++ fastfetch-2.8.6/src/logo/builtin.c	2024-05-02 09:29:47.115998751 +0200
@@ -3915,7 +3915,7 @@
 static const FFlogo T[] = {
     // T2
     {
-        .names = {"T2"},
+        .names = {"T2", "T2 SDE", "T2/Linux"},
         .lines = FASTFETCH_DATATEXT_LOGO_T2,
         .colors = {
             FF_COLOR_FG_WHITE,
